src/instance.ts

```diff
@@ -6,6 +6,8 @@ import { robloxLogObject, robloxActionObject } from "./types/robloxTypes";
 
 import NetworkComponent from "./components/networkComponent";
 import TerminalComponent from "./components/terminalComponent";
+import { ChildProcess, spawn } from "child_process";
+import path = require("path");
```

Comment:
I dislike this implementation, is there not a way for TypeScript to import 'path' like the above declarations? 

if not, add an empty line between the normal imports & this import as their is a clear difference between the two, the code should reflect that.

Reply:
Taken into account (https://github.com/4x8Matrix/OutSi-TS-Plugin/pull/12/commits/15382d8b796c43ac213bcdf96f51e9e3fc0cb128)!